src/bonbast/server.py

```diff
@@ -87,3 +90,61 @@ def get_prices_from_api(token, dont_raise_error=False):
         raise SystemExit(err)
 
     return r.json()
+
+
+def get_graph_data(start_date, end_date, currency, dont_raise_error=False):
```

Comment:
Please add parameters and return type

Reply:
ok, i will do your requests and commit again.